src/server/scripts/Spells/spell_quest.cpp

```diff
@@ -2482,6 +2482,62 @@ class spell_q13665_q13790_bested_trigger : public SpellScriptLoader
         }
 };
 
+// herald of war
+class spell_59064_portal_orgrimmar : public SpellScriptLoader
```

Comment:
can spell_59064_portal_orgrimmar  / spell_59439_portal_undercity  in a gen script for both move?